front/src/pages/room/style.tsx

```diff
@@ -0,0 +1,102 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.div`
+  border: 1px solid black;
+  box-sizing: border-box;
+  width: 360px;
+  height: 100%;
+  -webkit-box-shadow: 0px 1px 16px 2px #000000;
+  box-shadow: 0px 1px 16px 2px #000000;
+`;
+export const NameField = styled.div`
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid black;
+  width: 100%;
+  text-align: center;
+`;
+export const BackBtn = styled.button`
+  border: none;
+  background: none;
+`;
+export const ChatBox = styled.div`
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
+`;
+
+export const MessageBox = styled.div`
+  overflow: scroll;
+
+  width: 100%;
+  height: 500px;
+`;
+
+export const NameBox = styled.span<{ me?: boolean }>`
+  display: flex;
+  justify-content: ${(props) => (props.me ? 'flex-end' : null)};
+`;
+
+export const MsgBox = styled.span<{ me?: boolean }>`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  padding: 10px;
+  border-radius: ${(props) =>
+    props.me ? '12px 0px 12px 12px' : '0 12px 12px 12px'};
+
+  background: ${(props) =>
+    props.me ? 'rgb(25, 134, 252)' : 'rgb(241, 241, 243)'};
```

Comment:
같은 props 사용해서 여러 속성을 스타일링하고싶을때 ```css``` 함수 사용해도 좋을 것 같아요!

```suggestion
  ${({me}) =>
    me ? 
      css`
        border-radius: 12px 0px 12px 12px;
        background: rgb(25, 134, 252);
      `
      :
      css`
        border-radius: 0 12px 12px 12px;
        background: rgb(241, 241, 243);
      `
    }
```

Reply:
오 대박 저 이거 몰랐는데 너무 좋네요